rllib/utils/tests/test_checkpoint_utils.py

```diff
@@ -32,44 +41,164 @@ def test_get_checkpoint_info_v0_1(self):
             self.assertTrue(info["state_file"] == algo_state_file)
             self.assertTrue(info["policy_ids"] is None)
 
-    def test_get_checkpoint_info_v1_0(self):
-        # Create a simple (dummy) v1.0 Algorithm checkpoint.
-        with tempfile.TemporaryDirectory() as checkpoint_dir:
-            # algorithm_state.pkl
-            algo_state_file = os.path.join(checkpoint_dir, "algorithm_state.pkl")
-            Path(algo_state_file).touch()
-            # 2 policies
-            pol1_dir = os.path.join(checkpoint_dir, "policies", "pol1")
-            os.makedirs(pol1_dir)
-            pol2_dir = os.path.join(checkpoint_dir, "policies", "pol2")
-            os.makedirs(pol2_dir)
-            # policy_state.pkl
-            Path(os.path.join(pol1_dir, "policy_state.pkl")).touch()
-            Path(os.path.join(pol2_dir, "policy_state.pkl")).touch()
+    def test_get_checkpoint_info_v1_1(self):
```

Comment:
With msgpack being allowed, we up'ed the checkpoint version from 1.0 to 1.1.
Older versions of RLlib will still be able to read 1.1 checkpoints as long as they don't have msgpack as their "format".
Newer versions of RLlib will continue to be able to handle 1.0 checkpoints (these are always pickle-based).